csr/csr.go

```diff
@@ -64,25 +64,26 @@ func VerifyCSR(ctx context.Context, csr *x509.CertificateRequest, maxNames int,
 	if len(csr.EmailAddresses) > 0 {
 		return invalidEmailPresent
 	}
-	if len(csr.IPAddresses) > 0 {
-		return invalidIPPresent
+	if len(csr.URIs) > 0 {
+		return invalidURIPresent
 	}
 
-	// NamesFromCSR also performs normalization, returning values that may not
-	// match the literal CSR contents.
+	// NamesFromCSR and FromCSR also perform normalization, returning values
+	// that may not match the literal CSR contents.
 	names := NamesFromCSR(csr)
```

Comment:
With the introduction of idents.FromCSR below, there are now only two callers of NamesFromCSR (here and ca.go:557), both of which _only_ care about the CN field, ignoring the SANs.

I think this PR should also include stripping NamesFromCSR down into CNFromCSR, or leaving behind a TODO to that effect.